src/cdk-experimental/tree/control/base-tree-control.ts

```diff
@@ -12,6 +12,8 @@ import {TreeControl} from './tree-control';
 /** Base tree control. It has basic toggle/expand/collapse operations on a single data node. */
 export abstract class BaseTreeControl<T> implements TreeControl<T> {
 
+  parentMap: Map<T, T> = new Map<T, T>();
```

Comment:
Add description for this? Is this meant to be a public API? It would also be good to explain why parents are stored in a map. 

Also, if you don't need to enumerate the map, it would be better to use a `WeakMap`